src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/Tfm.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Server.IntegrationTesting
+{
+    public static class Tfm
+    {
+        public const string Net461 = "net461";
```

Comment:
Enum instead with ToString?

Reply:
No, it's an open set and it may even be case sensitive in some places.

Reply:
How often do we add  Tfms? And it shouldn't be case sensitive

Reply:
Every release. And you don't want to prevent one-off testing of other TFMs. E.g. doing a manual test against net471.

Reply:
This is only a helper to avoid having strings everywhere and prevent typoes.